hs-ucan/LICENSE

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Thanks for porting the license file! This library should be under Apache 2.

AGPL forces anyone that includes it to make their source available, which is a barrier to use, but good for us as e.g. Amazon can't just modify it and resell their tweaked version without making their changes also available. The "A" portion in particular is essentially a protection for binaries.

Since we're extracting the UCAN bits out in a hope that it will be widely used, including in closed source, so we want something more permissive. Apache 2 is the preferred choice, as essentially MIT updated with a patent covenant. Relicensing to a more permissive license is a thing that we can do.